src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/get_interior_vehicle_data_request_test.cc

```diff
@@ -122,7 +123,8 @@ class GetInteriorVehicleDataRequestTest
                                         mock_rpc_service_,
                                         mock_hmi_capabilities_,
                                         mock_policy_handler_,
-                                        mock_allocation_manager_);
+                                        mock_allocation_manager_,
+                                        mock_interior_data_cache_);
```

Comment:
@LuxoftAKutsan Why do you use mock_interior_data_cache_ only here? Should't be any EXPECTATIONS for this mock object added?

Reply:
@AByzhynar will be added